ding/policy/bc.py

```diff
@@ -58,6 +58,7 @@ def default_model(self) -> Tuple[str, List[str]]:
                 min=-0.5,
                 max=0.5,
             ),
+            normalize_states=False,
```

Comment:
move this field to policy common area, it is used in both collect and eval

Reply:
I will move it to the `cfg.env` and make it into a wrapper for d4rl environment.